platform/golden/cfg/preseed-debian-11.cfg

```diff
@@ -1,95 +1,109 @@
 d-i keyboard-configuration/xkb-keymap select us
-d-i console-keymaps-at/keymap select us
-d-i console-setup/ask_detect boolean false
-d-i debconf/frontend select Noninteractive
 
-####################
-## Network defaults
+
+####
+## B.4.2 Network configuration
 
 d-i netcfg/choose_interface select auto
 d-i netcfg/get_hostname string golden
-d-i netcfg/get_domain string local
-d-i anna/choose_modules string network-console
+d-i netcfg/get_domain string artificialwisdom.cloud
+
 
-###################
-## Mirror defaults
+####
+## B.4.4 Mirror settings
 
 d-i mirror/country string manual
 d-i mirror/http/hostname string http.us.debian.org
 d-i mirror/http/directory string /debian
 d-i mirror/http/proxy string
-d-i apt-setup/use_mirror boolean true
 
-#################
-## Time defaults
-
-d-i clock-setup/utc boolean true
-d-i time/zone string UTC
-d-i clock-setup/ntp boolean true
 
-##################
-## Login defaults
+####
+## B.4.5 Account setup
 
 d-i passwd/root-login boolean false
-d-i passwd/user-fullname string Artificial Wisdom
-d-i passwd/user-default-groups string users
-d-i passwd/user-default-groups string sudo
+d-i passwd/user-fullname string ArtificialWisdom
+d-i passwd/username string packer
+d-i passwd/user-password password insecure
+d-i passwd/user-password-again password insecure
+d-i passwd/user-default-groups string users sudo
 
 
-###################
-## Select packages
+####
+## B.4.6 Clock and time-zone setup
 
-tasksel tasksel/skip-tasks  string standard
-tasksel tasksel/first multiselect openssh-server
-popularity-contest popularity-contest/participate boolean false
+d-i clock-setup/utc boolean true
+d-i time/zone string UTC
+d-i clock-setup/ntp boolean true
 
-d-i pkgsel/include string curl wget openssh-server neovim sudo cloud-init
-d-i pkgsel/language-pack-patterns   string
-d-i pkgsel/install-language-support boolean false
-d-i pkgsel/upgrade select full-upgrade
 
-###########
-## Use XFS
+####
+## B.4.7 Partitioning
+## Our cloud requires the use of xfs
 
 d-i partman/default_filesystem string xfs
 d-i partman-auto/method string regular
 
-###################
-## Volume defaults
+
+####
+## B.4.7.aw0 Volume defaults
 
 d-i partman-auto/choose_recipe select atomic
 d-i partman-partitioning/confirm_write_new_label boolean true
 d-i partman/choose_partition select finish
 d-i partman/confirm boolean true
 d-i partman/confirm_nooverwrite boolean true
 
-####################
-## Additional media
-d-i apt-setup/cdrom/set-first boolean false
-d-i apt-setup/cdrom/set-next boolean false
-d-i apt-setup/cdrom/set-failed boolean false
-d-i apt-setup/non-free boolean false
-d-i apt-setup/contrib boolean false
 
-#######################
-## Bootloader defaults
+####
+## B.4.7.aw1 Boot defaults
 
-d-i grub-installer/only_debian boolean true
-d-i grub-installer/bootdev  string default
+d-i partman-efi/non_efi_system boolean true
+d-i partman-partitioning/choose_label select gpt
+d-i partman-partitioning/default_label string gpt
```

Comment:
does gpt require an MBR? I notice the disk has an MBR.

Reply:
yes, the MBR is where the GPT signature is supposed to be written (IIRC)

Reply:
reading the specifications of UEFI, MBR is not required when GPT is specified, with a complaint bootloader. Many past bootloaders were, however, non-compliant and they needed the MBR to recognize the disk.

Current bootloaders on modern hardware are compliant. However, there is no way, that is easy, to disable the writing of the MBR to the disk.

TLDR; the question is not relevant, thought i'd share what I learned.